Makefile

```diff
@@ -40,18 +44,22 @@ deps:
 	go get -u github.com/goadesign/gorma
 
 generate: $(DESIGNS)
-	goagen bootstrap -d ${PACKAGE_NAME}/${DESIGNDIR}
-	goagen js -d ${PACKAGE_NAME}/${DESIGNDIR} -o assets/ --noexample
-	goagen gen -d ${PACKAGE_NAME}/${DESIGNDIR} --pkg-path=github.com/goadesign/gorma
-	go-bindata-assetfs -debug assets/...
-	godep get
+	$(GOPATH)/bin/goagen bootstrap -d ${PACKAGE_NAME}/${DESIGNDIR}
```

Comment:
It's ok if we assume that there is one global GOPATH. I don't agree that this should be the case. We might have multiple workspaces with go code.